eng/pipelines/coreclr/runtime-nativeaot-outerloop.yml

```diff
@@ -162,6 +162,7 @@ extends:
         parameters:
           jobTemplate: /eng/pipelines/common/global-build-job.yml
           helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+          helixQueueGroup: cet
           buildConfig: Checked
           platforms:
           - windows_x64
```

Comment:
CET is a windows-x64 specific thing. I assume this does not apply to the linux_arm and other platform entries below?

Reply:
Yes, it doesn't apply to the other platforms